README.md

```diff
@@ -90,6 +90,16 @@ This script does the following steps for you:
  - create a TAB (tock application bundle)
  - if you have a nRF52-DK board connected: flash this TAB to your board (using tockloader)
 
+If you would like your `main()` to be `async`, you can wrap it using
+`libtock::async_main!`:
+```rust
+#![no_std]
+
+libtock::async_main!(async_main);
```

Comment:
For consistency with tokio I would like an attribute macro more than a macro_rules macro.